### PR TITLE
rootfs-configs.yaml: Add bullseye-igt rootfs image

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Add bullseye-igt rootfs configuration

Signed-off-by: Lakshmipathi Ganapathi <lakshmipathi.ganapathi@collabora.com>

### CODE REVIEW

**Comment by Lakshmipathi** [comment]
rootfs job: https://bot.staging.kernelci.org/job/laks/job/rootfs-builder/46/console
LAVA job: https://lava.collabora.co.uk/scheduler/job/4952270/definition

**Review by gctucker — APPROVED** [review]
LGTM, same pass/fail results as with the Buster rootfs:
https://linux.kernelci.org/test/plan/id/618e9e26acf9297aa6335918/

File: config/core/rootfs-configs.yaml [hunk 1]
```diff
@@ -51,6 +51,49 @@ rootfs_configs:
...
     test_overlay: "overlays/baseline"
 
+  bullseye-igt:
+    rootfs_type: debos
+    debian_release: bullseye
+    arch_list:
+        - amd64
+        - arm64
+        - armhf
+        - mips
```
Comment (gctucker) [inline config/core/rootfs-configs.yaml:61]: We know there's a problem with the plain bullseye build for mips, and as far as I know there's no platform with this architecture running IGT tests.  So maybe we could just drop it for bullseye-igt?

Comment (Lakshmipathi) [reply config/core/rootfs-configs.yaml]: Okay, dropped `mips` arch from this PR

**Review by gctucker — APPROVED** [review]

**Pull request merged**